src/main/java/com/google/sps/data/RestrictTimesCriteria.java

```diff
@@ -43,7 +43,7 @@ public float preferenceScore(Schedule schedule) {
     List<TimeRange> times = new ArrayList<TimeRange>();
 
     for(ScheduledCourse course : courses) {
-      times.addAll(course.getSection().getMeetingTimes());
+      times.addAll(course.getlectureSection().getMeetingTimes());
```

Comment:
Are you intending to not take into consideration a course's lab/recitation meeting times as well?

Reply:
I will add that check, after I pull from the parent branch.

Reply:
Added the labSections [here](https://github.com/googleinterns/step78-2020/pull/33/commits/b444eee8391730e2d7056eb1f646cb40ebac30b6)